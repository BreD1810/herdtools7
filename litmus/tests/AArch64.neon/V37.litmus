AArch64 V37
(* Test Neon final state checking
   LD1 multiple structure *)
{
x = 22;
0:X0=x;
}
 P0                         ;
 MOV W1,#1                  ;
 STR W1,[X0,#4]             ;
 MOV W1,#5                  ;
 STR W1,[X0,#8]             ;
 MOV W1,#44                 ;
 STR W1,[X0,#12]            ;
 LD1 {V0.4S},[X0]           ;
forall(0:V0.S = {22,1,5,44})
