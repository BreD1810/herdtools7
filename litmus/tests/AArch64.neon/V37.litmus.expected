Test V37 Required
Histogram (1 states)
1000000:>0:V0.S={22,1,5,44};
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (0:V0.S={22,1,5,44}) is validated
Hash=fb0dd67448de5f0e966fcdc4d6b74962
Observation V37 Always 1000000 0
